packages/framework/src/Model/Customer/DeliveryAddressFacade.php

```diff
@@ -52,14 +52,15 @@ public function edit(int $deliveryAddressId, DeliveryAddressData $deliveryAddres
 
     /**
      * @param \Shopsys\FrameworkBundle\Model\Customer\DeliveryAddressData $deliveryAddressData
-     * @return \Shopsys\FrameworkBundle\Model\Customer\DeliveryAddress
+     * @return \Shopsys\FrameworkBundle\Model\Customer\DeliveryAddress|null
      */
-    public function create(DeliveryAddressData $deliveryAddressData): DeliveryAddress
+    public function createIfAddressFilled(DeliveryAddressData $deliveryAddressData): ?DeliveryAddress
     {
         $deliveryAddress = $this->deliveryAddressFactory->create($deliveryAddressData);
```

Comment:
This should be changed to early return like:
```
if (!$deliveryAddressData->addressFilled) { 
    return null;
}

Reply:
hm that is debatable - you can not be sure  $this->deliveryAddressFactory->create does not return null (the internal logic of the method might change)

Reply:
fixed